code/clue.js

```diff
@@ -9,51 +9,215 @@ const mrGreen = {
+
+
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
-  name: 'rope',
+  name: 'Rope',
+  weight: 8
+}
+const knife = {
+  name: 'Knife',
+  weight: 2
+}
+const candlestick = {
+  name: 'Candlestick',
+  weight: 1
+}
+const dumbbell = {
+  name: 'Dumbbell',
+  weight: 15
+}
+const poison = {
+  name: 'Poison',
+  weight: 0
+}
+const axe = {
+  name: 'Axe',
   weight: 10
 }
+const bat = {
+  name: 'Bat',
+  weight: 7
+}
+const trophy = {
+  name: 'Trophy',
+  weight: 4
+}
+const pistol = {
+  name: 'Pistol',
+  weight: 3
+}
 
 // THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// GROUPS FOR ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS
 
 const suspects = [
   mrGreen,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  colMustard,
   mrsWhite
-  // ...  and the rest
 ]
 
-const weapons = []
+const weapons = [rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
 
-const rooms = []
+const rooms = ["Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// OBJECT THAT KEEPS THE MYSTERY.
+
+const mystery = {
+  killer: "",
+  weapon: "",
+  room: ""
+}
 
 // This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
+
+  // This will randomly select a killer from the suspects.
   mystery.killer = randomSelector(suspects)
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
-  document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  // This will change color of the card to the one connected to the chosen killer 
+  document.getElementById('killerCard').style.borderColor = mystery.killer.color
+
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerAge').innerHTML = `Age ${mystery.killer.age}`
+  document.getElementById('killerOccupation').innerHTML = `${mystery.killer.occupation}`
+
+  console.log(mystery.killer.image);
```

Comment:
Delete all console.logs when you are done with the project. We use them to make sure that everything works and under development, but if you have submitted the project as a pull request you are done with it :) 
```suggestion
```